xbstrap/mirror/__init__.py

```diff
@@ -21,15 +21,16 @@ def do_update(args):
         plan.dry_run = True
     if args.paranoid:
         plan.paranoid = True
-    if args.keep_going:
-        plan.keep_going = True
 
     # We always want to update mirrors.
     if not args.no_check:
         plan.check = True
     if not args.no_update:
         plan.update = True
 
+    # always do as much as possible
+    plan.keep_going = True
```

Comment:
this is an unrelated change, whoops, but I think it should be kept in. OK for splitting off into it's own commit?

Reply:
Why is that done?

Reply:
so that one mirror failing to pull won't stop others from updating

Reply:
Well, if you want that behavior, why not pass the existing command line parameter?

Reply:
oh, I didn't see that it's in the main parser